packages/lib/src/allTimeZones.ts

```diff
@@ -0,0 +1,110 @@
+const allTimeZones = [
```

Comment:
Let's keep it simple :)

```suggestion
const timeZones = [
```